src/tilemap.rs

```diff
@@ -43,6 +43,26 @@ fn spawn_chunk(commands: &mut Commands, asset_server: &AssetServer, chunk_pos: I
             });
         }
     }
+
```

Comment:
Add a comment why we need to keep these comment lines

Reply:
done